components/core/src/clp_s/CommandLineArguments.cpp

```diff
@@ -249,15 +269,42 @@ CommandLineArguments::parse_arguments(int argc, char const** argv) {
                 print_search_usage();
 
                 std::cerr << "Examples:" << std::endl;
-                std::cerr << "  # Search archives-dir for logs matching a KQL query" << std::endl;
-                std::cerr << "  " << m_program_name << " s archives-dir kql-query" << std::endl;
+                std::cerr << "  # Search archives-dir for logs matching a KQL query"
+                             R"( "level: INFO" and output to stdout)"
+                          << std::endl;
+                std::cerr << "  " << m_program_name << R"( s archives-dir "level: INFO")"
+                          << std::endl;
                 std::cerr << std::endl;
 
+                std::cerr << "  # Search archives-dir for logs matching a KQL query"
+                             R"( "level: INFO" and output to MongoDB)"
+                          << std::endl;
+                std::cerr << "  " << m_program_name
+                          << " s --mongodb-uri mongodb://127.0.0.1:27017/test --mongodb-collection "
+                             R"(test archives-dir "level: INFO")"
+                          << std::endl;
+
                 po::options_description visible_options;
                 visible_options.add(general_options);
+                visible_options.add(mongodb_options);
                 std::cerr << visible_options << '\n';
                 return ParsingResult::InfoCommand;
             }
+
+            if (false == m_mongodb_uri.empty() || false == m_mongodb_collection.empty()) {
```

Comment:
I think you could make this slightly more readable by swapping the if and else blocks.